packages/wallets/brave/README.md

```diff
@@ -0,0 +1,5 @@
+# `@solana/wallet-adapter-bravewallet`
```

Comment:
```suggestion
# `@solana/wallet-adapter-brave`
```

Reply:
fixed in https://github.com/solana-labs/wallet-adapter/pull/445/commits/db4c9ed630324c2db52183bebd76b75692bbd016